### ISSUE TITLE
Add Training Data 

### ISSUE THREAD

**Issue by ritwik12** [ritwik12/Celestial-bodies-detection#35]
There are few data available for `Uranus`, `Mars` and others. Having a good amount of training data increases accuracy also. Uranus is having only 20 images and for this model it should be at least 25 for better efficiency.

**Comment by kannandreams** [comment]
I worked to add few uranus images. please can you review the pull request and approve.
https://github.com/ritwik12/Celestial-bodies-detection/pull/36

**Comment by ritwik12** [comment]
@kannandreams  Sure, Reviewed Please check

**Comment by ferreiramonique** [comment]
Hi @ritwik12 ! I've added some training data for `Mars`, `Neptune` and `Uranus` (PR #62 for your approval).

**Comment by ritwik12** [comment]
@ferreiramonique Thanks!!

**Comment by varunpusarla** [comment]
Hey. Do you need more training data ? If yes, I'd like to contribute.

**Comment by ritwik12** [comment]
@varunpusarla Yes, we do need more but we need authentic and accurate images if you can help with that.

**Comment by SambhaviPD** [comment]
@ritwik12 - Is this issue still open? I checked the test data folder, I assume images on similar lines should be okay. Approximately how images is required?

**Comment by ritwik12** [comment]
@SambhaviPD There is never a limit to data, the more the images the better it is. But it would be better to have good and clear images only and not the animated or computerized ones.

**Comment by SambhaviPD** [comment]
@ritwik12 - Sure, let me first grab a few and submit a PR. Let know about the quality and I shall add more based on your feedback.
